src/pages/SignUpPage.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import NavBar from '../components/NavBar';
-import SignUp from '../components/SignUp';
+import AuthForm from '../components/AuthForm';
 
 function SignUpPage () {
   return (
     <div>
       <NavBar />
-      <SignUp />
+      <AuthForm
```

Comment:
`SignUp` and `SignIn` components were using too many share-able codes. They have been deleted and `AuthForm` is now being directly called in `SignUpPage` as well as `SignInPage` for efficiency & simplicity